src/tools/web/elektrad/README.md

```diff
@@ -1,30 +1,25 @@
 # elektra-web/elektrad
 
-A server that provides an [HTTP API](http://docs.elektrad.apiary.io/) to access
-Elektra remotely, built using [express](https://expressjs.com/).
+A server that provides an [HTTP API](http://docs.elektrad.apiary.io) to access
+Elektra remotely, built using [go](https://golang.org).
 
 ## Installation
 
-- first, install [libelektra](https://libelektra.org/)
-- now run `npm install`
+- make sure you have go (>1.11) installed.
+- install [libelektra](https://libelektra.org/), make sure the package-config env variable is set correctly.
+- now run `go build` in the elektrad folder with go module enabled (GO111MODULE=on).
 
 ## Running
 
-```
-npm start
-```
-
-to run in verbose mode (full logs):
+The output of `go build` is a binary, you can simply run it with:
 
 ```
-npm run start:verbose
+./elektrad
```

Comment:
Where is the binary located? Say it both for build directory and when installed.

Reply:
> Thank you for this PR! Docu is very sparse. Please also make sure that the jenkins build job succeeds (which is at the moment unfortunately a bit of a hassle).

I think the failing build had a network hiccup - but I'm not sure how to retry the job?

Reply:
`${build_folder}/src/tools/web/elektrad/elektrad`
and
`${install_folder}/tool_data/web/elektrad`, e.g. `/usr/local/share/elektra/tool_data/web/elektrad`